rdr_service/services/gcp_utils.py

```diff
@@ -821,16 +821,15 @@ def gcp_deploy_app(project, config_files: list, version: str = None, promote: bo
 
     return True
 
-def gcp_restart_instances(project=None):
+def gcp_restart_instances(self):
```

Comment:
Changing this to require an object with a `gcp_env` property makes this less flexible and isn't consistent with the other functions in this file.  Also, does this function restart instances for all services or only for the default service?  Can we use this function to only restart the cron service instances?

Reply:
I wouldn't say its required, if self does not have `gcp_env` it calls the `get_current_project` function. But I see your point and I'm happy to change it to better reflect other functions in the module. As it is this function only restarts default, I didn't see a reason to allow other services but I can add that if you'd like.

Reply:
It wouldn't hurt to allow it to restart other services besides the default service.